.github/workflows/helm-tests.yml

```diff
@@ -109,7 +109,7 @@ jobs:
             sleep 1
           done
           echo "vector service has started, moving on..."
-          while [[ -z "$(kubectl get pod  | grep "^stac-$RELEASE_NAME-.*$" | cut -d' ' -f1 | xargs -I{} kubectl logs pod/{} | grep "GET /_mgmt/ping" | head -n 1)" ]]; do
+          while [[ -z "$(kubectl get pod  | grep "^stac-$RELEASE_NAME-.*$" | cut -d' ' -f1 | xargs -I{} kubectl logs pod/{} | grep "GET /stac/_mgmt/ping" | head -n 1)" ]]; do
```

Comment:
I would love to understand why the other don't need this... but I'll just put that on my infinite todo list

Reply:
The other also need it, after updates...! They now all run on:

* `/raster`
* `/vector`
* `/stac`